br/pkg/task/restore.go

```diff
@@ -317,10 +334,10 @@ func IsStreamRestore(cmdName string) bool {
 }
 
 // RunRestore starts a restore task inside the current goroutine.
-func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConfig) error {
+func RunRestore(c context.Context, g glue.Glue, cmdName string, cfg *RestoreConfig) (uint64, error) {
```

Comment:
I think you can add a defer function
if err != nil then return 0, else return endversion.